pkg/kubectl/bind/plugin/bind.go

```diff
@@ -181,17 +168,22 @@ func (b *BindOptions) Run(ctx context.Context, urlCh chan<- string) error {
 			return err
 		}
 	}
+
+	auth := authenticator.NewLocalhostCallbackAuthenticator(10 * time.Minute)
+	err = auth.Start()
+	fmt.Fprintf(b.Options.ErrOut, "\n\n")
```

Comment:
why did we have this in the first place? (left unchanged for the refactor)

Reply:
To separate it from other output I guess.